packages/ckan/.gitignore

```diff
@@ -0,0 +1,9 @@
+.DS_Store
+node_modules
+/dist
+.env
+
+# Log files
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```

Comment:
Ah, probably leftovers from old repo?

Reply:
I will remove entries listed in the root one, but keep specific ones for this plugin.